plugins/katello/nightly/user_guide/srpm_support/index.md

```diff
@@ -0,0 +1,169 @@
+
+```jsonld=
+[
+    {
+        "uploads"=>[{"id"=>"3896b10c-2694-40d5-b380-a45065d735ca",
+            "name"=>"garmindev-0.3.4-9.el7.src.rpm",
+            "size"=>91660,
+            "checksum"=>"f9b9ec9735fd05e1153ba52e8f027c466f8cb0d2c76ca5ebcc6b982f69ebb560"}],
+        "publish_repository"=>true,
+        "sync_capsule"=>true,
+        "content_type"=>"srpm",
+        "api_version"=>"v2",
+        "id"=>"1",
+        "repository"=>{"content_type"=>"srpm"}
+    }
+]
+```
+
+To see all of the available options to use with the new source RPM API, click [here](https://theforeman.org/plugins/katello/3.13/api/apidoc/v2/srpms.html)
+
+## Hammer additions
+
+* The Hammer CLI tool has two new commands: `hammer srpm list` and `hammer srpm info`.
+
+* The `hammer srpm list` command lists source RPMS by default across the entire Katello server. This can be filtered down by organization, product, repository, content-view and lifecycle environment.
+
+* The `hammer srpm info` command gathers information about a source RPM. Returning ID, Name, Version, Architecture, Epoch, Release, Filename, and Description.
+
+## Sample workflow to create a source RPM repository, upload a source RPM, and list a source RPM
+
+### Create a product and repository
+
+Since the feature is only available in the CLI/API currently, I will be using Hammer for this workflow.
+
+First we want to create a product:
+
+```bash
+# hammer product create --name Zoo --organization-id 1
+
+Product created.
+```
+
+Next we want to grab our product id for the repository creation step:
+
+```bash
+# hammer product list --organization-id 1
+
+---|------|-------------|----------------------|--------------|-----------
+ID | NAME | DESCRIPTION | ORGANIZATION         | REPOSITORIES | SYNC STATE
+---|------|-------------|----------------------|--------------|-----------
+1  | Zoo  |             | Default Organization | 0            |
+---|------|-------------|----------------------|--------------|-----------
+```
+
+Next we will create our repo:
+
+```bash
+# hammer repository create --product-id 1 --organization-id 1 --content-type yum --name source_rpms --publish-via-http yes
+
+Repository created.
+```
+
+Now we want to grab our repo id for the upload step:
+
+```bash
+# hammer repository list
+
+---|-------------|---------|--------------|----
+ID | NAME        | PRODUCT | CONTENT TYPE | URL
+---|-------------|---------|--------------|----
+1  | source_rpms | Zoo     | yum          |
+---|-------------|---------|--------------|----
+```
+
+### Upload a source RPM into a repository
+
+Now that we have our repository, we will upload a source RPM using the following command:
+
+```bash
+hammer repository upload-content --content-type srpm --id 1 --product-id 1 --path /root/garmindev-0.3.4-9.el7.src.rpm
+
+Successfully uploaded file 'garmindev-0.3.4-9.el7.src.rpm'
+```
+
+Now we will check to make sure it uploaded correctly:
```

Comment:
This blurb casts doubt on whether it will upload correctly or not. We can frame it more positively for example: `Retrieving the repository info will show that the SRPM was uploaded. Note the content counts.`